python/plugins/processing/ui/DlgScriptEditor.ui

```diff
@@ -155,6 +156,17 @@
     <string>Ctrl+Shift+S</string>
    </property>
   </action>
+  <action name="actionLoadFromTemplate">
+   <property name="text">
+    <string>Load script from template...</string>
+   </property>
+   <property name="toolTip">
+    <string>oad script from template...</string>
+   </property>
+   <property name="shortcut">
+    <string>Ctrl+Shift+M</string>
```

Comment:
I'd drop this shortcut - I don't think it's warranted.

Reply:
Agree.. I had to edit the UI manually because I cannot add actions with Qt Designer, so I followed all the other actions *guidelines*